Dockerfile

```diff
@@ -3,12 +3,10 @@ FROM python:2.7-slim
 MAINTAINER bhearsum@mozilla.com
```

Comment:
Yeah, it works fine, but it's a PITA to do if you want to regularly test production-like containers.